### PR TITLE
Astana, Kazakhstan

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
I'll be adding them in separate PRs since the data model is not homogenous, so there will be issues related to specific files.

### CODE REVIEW

**Comment by simonkas** [comment]
I dont underdstsnd what is happening. My macbook crashed lastnight when i tried to reinstall sierrra, found maverickk under it, library was all script and imessage... Tried copying one by one to flssh, think I got all but got caught.. Am using another phone, iphone is not being controlled, my new iphone7 got stolen a weeks ago, got a new 6c but has same # as stolen i7. Not safe neither is this... Linked same wifi modem. Completely stuck.

**Comment by simonkas** [comment]
Maps from my condo complex, 4 large buildings, believe watched from one of the buildings.. Have pics but not onthis phone. Will hav to go back to iphone.. How can I be in touch and help, have tons of screen shots.. Where can u contact or i contact u and know it is u? Im not sure what to do anymore, on it foor yeats but alone and trying to learn, getting bad over my head.

**Comment by iandees** [comment]
(Removed spam comments)

File: sources/kz/asa-statewide.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    },
+    "data": "http://infopublic.pravstat.kz:8399/arcgis/rest/services/Kazakhstan/MapServer/25",
+    "attribution": "Комитет по правовой статистике и специальным учетам Генеральной прокуратуры Республики Казахстан",
+    "type": "ESRI",
+    "language": "ru",
+    "conform": {
+        "type": "geojson",
+        "city": "Астана",
+        "number": "NUMBER_1",
+        "street": "NAME_STREETS"
```
Comment (thatdatabaseguy) [inline sources/kz/asa-statewide.json]: It seems to me that the ARHIV_STREET_1 field has better street names (usually the same name with ул. in front but maybe there are other differences).

**Comment by simonkas** [comment]
Please tell me where or how to communicate with you, my mac is crashed. My iphone is compromised but I still have some control a well as many screen shots.... But I dont know whos who to trust. It is being charged now but u can leave message at this email and I will change back to my phone when i can. I am in o er my head, dont understand anymore. Cant tell who is real site or fake. Need help desperately, starting to feel threatened. Been battleing thos for at least 6 - 8 years alone, https://password.mail.com/passwordrecovery/wicket/page?1&srttkn=775adaf5-f442-49ad-90d4-6631423cf6dThe full stack trace of the root cause ...???🔐😏 this just popped up on my message???noone will help me.cell 4165605344 / email simona.f@ email.com. please help me. I am unable to go use devices any longer.... Being watched.

**Comment by migurski** [comment]
(more spam comments deleted, spammy user reported)

**Comment by iandees** [comment]
Going with this for now!

**Pull request merged**